pkg/gin-router-wrapper/engine.go

```diff
@@ -0,0 +1,99 @@
+package gin_router_wrapper
```

Comment:
Maybe we can just call it as `engine` package.

Reply:
Well, the `gin_router_wrapper` does look kind of ugly.